app/containers/EosioProvider/saga.js

```diff
@@ -25,6 +25,7 @@ import { initEosioSuccess, initEosioError } from './actions';
 import { INIT_EOSIO, INIT_EOSIO_SUCCESS } from './constants';
 
 import validate from './validate';
+import { logout } from '../Logout/actions';
```

Comment:
I would use absolute import for importing from another directory

Reply:
Fixed